hist/hist/inc/TH1.h

```diff
@@ -137,6 +145,7 @@ class TH1 : public TNamed, public TAttLine, public TAttFill, public TAttMarker {
                                Option_t * opt, Bool_t doerr = kFALSE) const;
 
    virtual void     DoFillN(Int_t ntimes, const Double_t *x, const Double_t *w, Int_t stride=1);
+   Bool_t    GetStatOverflowsBehaviour() const { return EStatOverflows::kNeutral == fStatOverflows ? fgStatOverflows : EStatOverflows::kConsider == fgStatOverflows; }
```

Comment:
Shouldn't the last one be `fStatOverflows` ?

Reply:
That is correct andrea, thanks! The test pass since the enumerator EStatOverflows is wise enough to mimic a bool for the kConsider and kIgnore constants...